tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -42,11 +42,20 @@ def setUp(self):
                 "auth.teams.list",  # app-level token
                 "oauth.access",
                 "oauth.v2.access",
+                "oauth.v2.exchange",
                 "oauth.token",
+                "openid.connect.token",
+                "openid.connect.userInfo",
                 "users.setActive",
                 "admin.conversations.getCustomRetention",
                 "admin.conversations.removeCustomRetention",
                 "admin.conversations.setCustomRetention",
+                # TODO: Slack Connect APIs
```

Comment:
I believe @srajiang will look at adding these when she gets back next week

Reply:
Great! Happy to be a reviewer for it

Reply:
Just for reference: https://github.com/slackapi/python-slack-sdk/issues/1058